test/integration/acceptance/transfer_asset_test.cpp

```diff
@@ -475,63 +358,24 @@ TEST_F(TransferAsset, BigPrecision) {
       }
     };
   };
+
   auto make_query = [this](std::string account_id) {
-    return proto::QueryBuilder()
+    return baseQry()
         .creatorAccountId(IntegrationTestFramework::kAdminId)
-        .createdTime(getUniqueTime())
         .getAccountAssets(account_id)
-        .queryCounter(1)
         .build()
         .signAndAddSignature(kAdminKeypair)
         .finish();
   };
 
   IntegrationTestFramework(1)
       .setInitialState(kAdminKeypair)
-      .sendTx(makeUserWithPerms(kUser1, kUser1Keypair, kPerms, kRole1))
-      .skipProposal()
-      .skipBlock()
-      .sendTx(makeUserWithPerms(kUser2, kUser2Keypair, kPerms, kRole2))
-      .skipProposal()
-      .skipBlock()
-      .sendTx(proto::TransactionBuilder()
-                  .creatorAccountId(
-                      integration_framework::IntegrationTestFramework::kAdminId)
-                  .createdTime(getUniqueTime())
-                  .quorum(1)
-                  .createAsset(kNewAsset,
-                               IntegrationTestFramework::kDefaultDomain,
-                               precision)
-                  .build()
-                  .signAndAddSignature(kAdminKeypair)
-                  .finish())
-      .skipProposal()
-      .checkBlock([](auto &block) {
-        ASSERT_EQ(block->transactions().size(), 1) << "Cannot create asset";
-      })
-      .sendTx(proto::TransactionBuilder()
-                  .creatorAccountId(kUser1Id)
-                  .createdTime(getUniqueTime())
-                  .quorum(1)
-                  .addAssetQuantity(kUser1Id, kNewAssetId, kInitial)
-                  .build()
-                  .signAndAddSignature(kUser1Keypair)
-                  .finish())
-      .checkBlock([](auto &block) {
-        ASSERT_EQ(block->transactions().size(), 1) << "Cannot add assets";
-      })
-      .sendTx(proto::TransactionBuilder()
-                  .creatorAccountId(kUser1Id)
-                  .createdTime(getUniqueTime())
-                  .quorum(1)
-                  .transferAsset(
-                      kUser1Id, kUser2Id, kNewAssetId, kDesc, kForTransfer)
-                  .build()
-                  .signAndAddSignature(kUser1Keypair)
-                  .finish())
-      .checkBlock(
-          [](auto &block) { ASSERT_EQ(block->transactions().size(), 1); })
-      .sendQuery(make_query(kUser1Id), check_balance(kUser1Id, kLeft))
+      .sendTxAwait(makeFirstUser(), check(1))
+      .sendTxAwait(makeSecondUser(), check(1))
+      .sendTxAwait(create_asset, check(1))
+      .sendTxAwait(add_assets, check(1))
+      .sendTxAwait(make_transfer, check(1))
+      .sendQuery(make_query(kUserId), check_balance(kUserId, kLeft))
       .sendQuery(make_query(kUser2Id), check_balance(kUser2Id, kForTransfer))
```

Comment:
Despite the test does not fail, the query causes postgres' error:
```
[2018-06-21 22:45:45.544989785][th:20689][info] IntegrationTestFramework send query
[2018-06-21 22:45:45.552626695][th:20689][error] PostgresWsvQuery ERROR:  invalid input value for enum grantable_perm: ""
LINE 1: ...n@test' AND account_id = 'usertwo@test' AND permission = '';
                                                                    ^
```

Reply:
Will be resolved with IR-1464